x-pack/plugins/security_solution/common/endpoint/types/index.ts

```diff
@@ -843,6 +843,7 @@ type KbnConfigSchemaNonOptionalProps<Props extends Record<string, unknown>> = Pi
  */
 export interface PolicyConfig {
   windows: {
+    advanced: AdvancedFields;
```

Comment:
I'm not sure we should add this yet since it would adjust the policy config for 7.10. We need to make sure we handle this correctly when there are old policies that do not contain the fields..

Reply:
I suggest we don't include this as part of this PR.